invenio_rdm_pure/source/rdm/marc21_record.py

```diff
@@ -142,3 +151,14 @@ def add_unique_value(
         if not self.contains(datafield, subfield):
             datafield.subfields.append(subfield)
             self.datafields.append(datafield)
+
+    @staticmethod
+    def validate_marc21_xml_string(record: str) -> str:
```

Comment:
Use this method to test the Converter() output.

Reply:
Please see https://github.com/tu-graz-library/invenio-rdm-pure/pull/33/commits/f4c46027ab75a52fdf9ecd9d15d84bbd60aeb754, I hope this is what you meant.